certified-connectors/IA-Connect Session/apiDefinition.swagger.json

```diff
@@ -7724,6 +8408,79 @@
         "operationId": "GetCSVTextAsCollection"
       }
     },
+    "/FileManagement/WriteCollectionToCSVFile": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "summary": "Write to CSV file",
+        "description": "Creates a CSV file on the computer where the IA-Connect Agent is running. The CSV file is created from a collection (table) or from a table in JSON format.",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "WriteCollectionToCSVFile",
+            "description": "Inputs for IA-Connect WriteCollectionToCSVFile action",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "InputTable": {
+                  "type": "array",
+                  "title": "Properties",
+                  "description": "A table of data to write to the CSV file.",
+                  "items": {
+                    "type": "object"
+                  }
+                },
+                "InputTableJSON": {
+                  "type": "string",
+                  "title": "Input table JSON",
+                  "description": "A table of data to write to the CSV file, in JSON format."
+                },
+                "CSVFilePath": {
+                  "type": "string",
+                  "title": "Output CSV file path",
+                  "description": "The full path to the CSV file. This must be accessible from the session where the IA-Connect Agent is running."
+                },
+                "OutputEncoding": {
+                  "type": "string",
+                  "default": "UTF8",
+                  "title": "Output encoding",
+                  "description": "The text encoding format to use when writing to the CSV file. Supported encodings are UTF8, UTF7, Unicode and ASCII.",
+                  "enum": [
+                    "UTF8",
+                    "UTF7",
+                    "Unicode",
+                    "ASCII"
+                  ]
+                },
+                "Workflow": {
+                  "type": "string",
+                  "x-ms-visibility": "advanced",
+                  "default": "@{workflow()}",
+                  "title": "Workflow",
+                  "description": "Add the following expression here: workflow()"
+                }
+              },
+              "required": [
+                "CSVFilePath",
+                "Workflow"
+              ]
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {}

```

Comment:
@LukeMcGovern-UL Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Reply:
@sriyen-msft Thank you for reviewing our pull request - in this scenario the command returns no output and therefore has no output schema. We distinguish between success or failure by returning the relevant HTML code, 200 (success) or 400 (failure) which will result in an exception in the Power Automate Flow. 

Most of our actions do return data and all of these have an appropriate output schema defined.